assets/src/design-system/components/input/useNumericInput.js

```diff
@@ -47,7 +47,7 @@ export const useNumericInput = ({
    */
   const handleBlur = useCallback(
     (ev) => {
-      let newValue = parseInput(oldValue.current, options);
```

Comment:
@samwhale I've removed the `changeTracker` from this file, allowing the `isIndeterninate` be `true` when the value changes back to original as well.

This now only submits the value when it has changed as well.